hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -419,5 +419,30 @@
             - update_err: A shadow register encountered an update error.
             '''
     }
+    {
+      name: eviction_cg
+      desc: '''
+            Covers eviction with mp_region_cfgs for data and info regions.
+            Sample all 4 rd_buf status.
+            When each buffer hazard is set, capture the address stored in the buffer.
+            Then search from tb data base to see which region the address belong to.
+            After that record the config value (scrambe_en and ecc_en) of the region.
+            Use cross over buffer index, the operation to cause the eviction and
+            the region config values.
+            '''
+    }
+    {
+      name: fetch_code_cg
+      desc: '''
+            Covers whether dut is received valid / invalue key value from ral.exec
```

Comment:
This needs some clarification as well. The a_user.instr_type is a mubi4?

Regardless, the text needs at least these changes.
```suggestion
            Covers whether dut received valid or invalid key value from ral.exec.
```

Reply:
yes it is mubi4. Updated comment as suggested.